docs-website/src/pages/docs/_components/FeatureCardSection/featurecardsection.module.scss

```diff
@@ -0,0 +1,36 @@
+
+
+.feature {
```

Comment:
minor thing - the text in the feature cards being centered feels a bit strange - left aligned might look nicer for the header/subtitle text